extensions/amp-analytics/0.1/variables.js

```diff
@@ -173,6 +174,24 @@ function matchMacro(string, matchPattern, opt_matchingGroupIndexStr) {
   return matches && matches[index] ? matches[index] : '';
 }
 
+/**
+ * Returns a comma separated list of active branch experiment names and their
+ * values, or `null` if none are active.
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+ * @return {?string}
+ */
+function experimentBranchesMacro(ampdoc) {
+  const parent = ampdoc.getParent();
+  const win = parent ? parent.win : ampdoc.win;
+  const branches = getActiveExperimentBranches(win);
```

Comment:
instead of having the analytics code to find the top window, can we have `getActiveExperimentBranches` to return the experiments applied at top window. Since the experiments are going to be applied to child window anyway.

Reply:
done.